src/System.Resources.Reader/src/System.Resources.Reader.csproj

```diff
@@ -17,6 +18,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System\Resources\ResourceReader.cs" />
+    <Compile Include="System\Resources\StreamExtensions.cs" />
```

Comment:
Do we expect this could be useful elsewhere?  I'm wondering if it should be given a more descriptive name and put somewhere under src\Common\src.